src/main/scala/com/gu/ami/amiup/aws/PollDescribeStackStatus.scala

```diff
@@ -45,7 +45,7 @@ object PollDescribeStackStatus extends LazyLogging {
 
   private def getStackStatuses(stacks: Seq[Stack], client: CloudFormationAsyncClient)(implicit ec: ExecutionContext): EitherT[Future, String, Seq[Stack]] = {
     EitherT {
-      AWS.describeStacks(client).map { dsr =>
+      AWS.describeStacks(client, None).map { dsr =>
```

Comment:
We might consider filtering this request at the AWS end rather than in Scala by filtering the results of looking up all the stacks, similar to how you've done for YOLO mode. Not something to worry about in this PR though!